### DISCUSSION TITLE
Infinity indexing

### DISCUSSION THREAD

**Discussion by Dedsd** [photoprism/photoprism#1972] [Q&A] [answered]
I'm getting this infinity indexing and can't access the webui
```
time="2022-01-24T02:00:17Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2022-01-24T02:00:17Z" level=info msg="Visit https://docs.photoprism.app/funding/ to learn more."
time="2022-01-24T02:00:20Z" level=info msg="migrate: found no previous migrations"
time="2022-01-24T02:00:20Z" level=info msg="migrate: 20211121-094727 successful [49.273151ms]"
time="2022-01-24T02:00:20Z" level=info msg="migrate: 20211124-120008 successful [54.589638ms]"
time="2022-01-24T02:00:22Z" level=info msg="webdav: /originals/ enabled, waiting for requests"
time="2022-01-24T02:00:22Z" level=info msg="webdav: /import/ enabled, waiting for requests"
time="2022-01-24T02:00:22Z" level=info msg="http: starting web server at 0.0.0.0:2342"
time="2022-01-24T02:15:22Z" level=info msg="index: updating counts"
time="2022-01-24T02:15:22Z" level=info msg="index: updating covers"
time="2022-01-24T02:30:22Z" level=info msg="index: updating counts"
time="2022-01-24T02:30:22Z" level=info msg="index: updating covers"
time="2022-01-24T02:45:22Z" level=info msg="index: updating counts"
time="2022-01-24T02:45:22Z" level=info msg="index: updating covers"
time="2022-01-24T03:00:22Z" level=info msg="index: updating counts"
time="2022-01-24T03:00:22Z" level=info msg="index: updating covers"
time="2022-01-24T03:15:22Z" level=info msg="index: updating counts"
time="2022-01-24T03:15:22Z" level=info msg="index: updating covers"
time="2022-01-24T03:30:22Z" level=info msg="index: updating counts"
time="2022-01-24T03:30:22Z" level=info msg="index: updating covers"
```

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/troubleshooting/#connection-fails

There is no infinite indexing, it's just the background worker. Increase the log level and update to the latest version if you don't want to see it.

**Reply by Dedsd** [reply]
I'm using pure docker how could I increase the log level. I was using `sudo docker logs photoprism`.

**Reply by graciousgrey** [reply]
You need to change the value of PHOTOPRISM_LOG_LEVEL , e.g. to "warning" (https://docs.photoprism.app/getting-started/config-options/)

**Reply by lastzero** [reply]
Hewever, if you can't connect... You should reduce the log level to debug so you can figure out what is wrong.

**Comment by Genygo** [comment]
With pure docker you mean without docker-compose?
I recommend using docker-compose. it makes updating the container a lot easier. 
For your connection problem:
Did you bind port 2342 to your host?

**Reply by Dedsd** [reply]
What do you mean with  bind port 2342?

**Reply by lastzero** [reply]
See [Documentation](https://docs.photoprism.app/getting-started/docker/) `-p 2342:2342`:
```bash
docker run -d \
  --name photoprism \
  --security-opt seccomp=unconfined \
  --security-opt apparmor=unconfined \
  -p 2342:2342 \
  ...
```

**Reply by Dedsd** [reply]
Yeah, I'm using the default `-p 2342:2342` with no firewall or proxy

**Comment by Dedsd** [comment]
@lastzero Hi, I solved the problem of running. But now I tried to sync with nextcloud and It's very bugged. My originals directory don't show all the photos that are In my nextcloud folder, but photoprism webui shows me even less photos than the one in originals. Index like 4x times now.. Nothing in the logs, just this: 
```
started as root
running as root
photoprism start
time="2022-01-24T17:00:08Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2022-01-24T17:00:08Z" level=info msg="Visit https://docs.photoprism.app/funding/ to learn more."
```

**Reply by lastzero** [reply]
Did you already take a look at our Troubleshooting Checklists?

**Reply by Dedsd** [reply]
I Troubleshoot adding 6gb of swap. But still gives me this bug with sync.

**Reply by Dedsd** [reply]
before I was getting this error: 
```
runtime: out of memory: cannot allocate 536870912-byte block (1676804096 in use)
fatal error: out of memory
```

**Reply by Dedsd** [reply]
It needs more than 4gb of swap? I followed the docs but still are giving the fatal error: out of memory.

**Reply by lastzero** [reply]
Depends how big your images / files are...

**Reply by Dedsd** [reply]
Aprox: 1.8gb first time running the sync it already run out of memory

**Reply by lastzero** [reply]
What files? How many workers? What hardware specs? Shall we spend the day guessing everything?

**Reply by Dedsd** [reply]
1- Images and videos
2- 1
3- Raspberry pi 4 4gb(RAM) 256gb(STORAGE) 4.2gb swap
4- No, I'm sorry

**Comment by Dedsd** [comment, marked as answer]
I think the issue is similar to https://github.com/photoprism/photoprism/discussions/968 , but in my case I got just 11 photos showing in the web ui, compared to 30 photos that are in /originals. Also a video folder in nextcloud couldn't be indexed to /originals and I still don't know why.

**Reply by Dedsd** [reply]
No repeated photos

**Reply by lastzero** [reply]
File size limit?

**Reply by Dedsd** [reply]
Was actually the quality filter setting. I'm gonna wait for the videos to show up and if everything looks right I'll mark this as the answer. Thanks!

**Reply by Dedsd** [reply]
Oh, the videos could be the file size limit?

**Reply by lastzero** [reply]
See  `PHOTOPRISM_ORIGINALS_LIMIT`:

https://docs.photoprism.app/getting-started/config-options/#storage-folders
